cmd/serve.go

```diff
@@ -85,7 +85,7 @@ var serveCmd = &cobra.Command{
 			}
 
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged